text-generation-inference/server/text_generation_server/models/causal_lm.py

```diff
@@ -560,13 +648,37 @@ def __init__(
             else:
                 tokenizer.add_special_tokens({"pad_token": "[PAD]"})
 
+        kwargs = {
+            "use_cache": True,
+            "return_dict": True,
```

Comment:
Why do we need `return_dict`?

Reply:
it's from https://github.com/huggingface/optimum-habana/blob/main/text-generation-inference/server/text_generation_server/models/causal_lm.py#L587